src/webui/src/components/PackageSidebar/modules/Infos/style.scss

```diff
@@ -0,0 +1,22 @@
+@import '../../../../styles/variable';
+
+.infosModule {
+  li {
+    display: flex;
+    font-size: 14px;
```

Comment:
We don't have a font size variable here, but It is a good moment to create one instead hardcode font in this file. 
https://github.com/verdaccio/verdaccio/blob/master/src/webui/src/styles/variable.scss#L21

Reply:
All of the other modules in the sidebar seem to hard code the font size currently. Would it maybe better to create a separate pr and change all of them at once?

Reply:
Ok, make sense. I'll add a note in the projects section.